Makefile

```diff
@@ -74,6 +74,14 @@ gdbstub-test: $(BIN)
 	$(Q)tests/gdbstub.sh && $(call notice, [OK])
 endif
 
+# Import adaptive replacement cache
+ENABLE_ARCACHE ?= 1
```

Comment:
Don't add both `ENABLE_ARCACHE` and `ENABLE_ARCACHE_INFO`.
I would like to have a minimal ARC implementation along with a test program. Then, we can integrate the refined one into rv32emu core in further pull requests.

Reply:
There should no entry `ENABLE_ARCACHE`. That is, this pull patch only delivers ARC header and C implementation along with test program(s). You should tweak `Makefile` for the linkage of ARC test program(s).

Reply:
OK, I understand.

Reply:
The name `ARCACHE` sounds stupid since ARC stands for Adaptive Replacement Cache.

Reply:
This ARC implementation is meant be reusable for further JIT compilation. Its quality should be verified, and it is exactly why test programs should appear for validation purposes. The interface to ARC should be generic and straightforward without disclosing the details.